velox/exec/Driver.cpp

```diff
@@ -539,6 +539,7 @@ StopReason Driver::runInternal(
             curOperatorId_,
             kOpMethodIsBlocked);
         if (blockingReason_ != BlockingReason::kNotBlocked) {
+          VELOX_CHECK(future.valid());
```

Comment:
Thanks. 

Would it be possible to add an error message to clarify what's wrong? It would be nice to specify which operator returned incorrect result similar to how we do that in CALL_OPERATOR.

Reply:
Got it, fixed.